README.md

```diff
@@ -192,14 +192,14 @@ For testing. This generator repeats back the prompt it received.
 | promptinject | Implementation of the Agency Enterprise [PromptInject](https://github.com/agencyenterprise/PromptInject/tree/main/promptinject) work (best paper awards @ NeurIPS ML Safety Workshop 2022) |
 | realtoxicityprompts | Subset of the RealToxicityPrompts work (data constrained because the full test will take so long to run) |
 | snowball | [Snowballed Hallucination](https://ofir.io/snowballed_hallucination.pdf) probes designed to make a model give a wrong answer to questions too complex for it to process |
-| xss | Look for vulnerabilities the permit or enact cross-site attacks, such as private data exfilteration. |
+| xss | Look for vulnerabilities the permit or enact cross-site attacks, such as private data exfiltration. |
 
 ## Logging
 
 `garak` generates multiple kinds of log:
 * A log file, `garak.log`. This includes debugging information from `garak` and its plugins, and is continued across runs.
 * A report of the current run, structured as JSONL. A new report file is created every time `garak` runs. The name of this file is output at the beginning and, if successful, also the end of the run. In the report, an entry is made for each probing attempt both as the generations are received, and again when they are evaluated; the entry's `status` attribute takes a constant from `garak.attempts` to describe what stage it was made at.
-* A hitlog, detailing attempts that yielded a vulnerability (a 'hit')
```

Comment:
We safe the files as `{prefix}.hitlog.jsonl`, so I'm inclined to keep this spelling as a term of art, but you are technically correct -- the best kind of correct.

Reply:
Consider then expanding this similar to the `log file` entry above as:

```
* A hit log, `{run_id}.hitlog.jsonl`, detailing attempts that yielded a vulnerability (a 'hit')
```